data-raw/standard-format-data-prep/rst_catch_standard_format.Rmd

```diff
@@ -519,6 +528,28 @@ knights_rst_clean <- knights_landing_rst %>%
   group_by(fork_length, weight, species, run, run_method, lifestage, adipose_clipped, date) %>%
   summarize(count = sum(count, na.rm = T))
 
+knights_rst_clean_pre2006 <- knights_landing_pre2006 %>% 
+  # filter to only include data before 2006 (CAMP stores data 2006 to present)
+  filter(date < "2006-10-02") %>% 
+  select(-c(start_date, stop_date, location, fork_length_max_mm, fork_length_min_mm, cpue)) %>% 
+  rename(run = at_capture_run,
+         adipose_clipped = marked) %>% 
+  mutate(adipose_clipped = case_when(adipose_clipped == T ~ "Natural",
+                                     adipose_clipped == F ~ "Hatchery",
+                                     T ~ "Unknown"),
+         count = ifelse(is.na(count), 0, count),
+         # this likely happened from reshaping of data
+         remove = case_when(is.na(run) & species != "Steelhead" ~ "remove",
+                            species == "Steelhead" ~ "remove",
```

Comment:
Are we sure we ant to remove chinooks with no run? And steelhead? I know we went back and forth on this but I also am now thinking if we want to make this data as useful as possible to a wider audience we might as well leave in other species

Reply:
I added steelhead back in and also reran CAMP data for Knights to include all species. I will do the same for Feather and maybe Yuba if they have other species.

Reply:
Thank you!